plugins/auth-backend/src/service/router.ts

```diff
@@ -36,14 +36,17 @@ export async function createRouter(
   const router = Router();
   const logger = options.logger.child({ plugin: 'auth' });
 
-  const baseUrl = `${options.config.getString('backend.baseUrl')}/auth`;
+  const appOrigin = options.config.getString('app.baseUrl');
```

Comment:
Not sure if it was clear in the thread so wanna reiterate that the `app.baseUrl` needs to be parsed to extract only the origin. The baseUrl may contain path or even query parts that we don't want included in the origin.